scripts/build/companion_libs/350-newlib_nano.sh

```diff
@@ -207,10 +207,19 @@ ENABLE_TARGET_OPTSPACE:target-optspace
 *cc1plus:
 -idirafter %:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/include) %(newlib_nano_cc1plus)
```

Comment:
Should update `*cpp` and `*cc1plus` include directory as well so that we pull in the correct `newlib.h` under `include/newlib-nano` when `CT_NEWLIB_NANO_INSTALL_IN_TARGET=y`.

```suggestion
*cc1plus:
-idirafter %:getenv(GCC_EXEC_PREFIX ../../${CT_TARGET}/include/newlib-nano) %(newlib_nano_cc1plus)
```